### DISCUSSION TITLE
Focus log window if already opened

### DISCUSSION THREAD

**Discussion by CoffiDev** [johnlindquist/kit#1193] [Q&A]
I have a small script that I use to get some info on my windows

```
await showLogWindow()

const items = await getWindowsBounds()

await log(JSON.stringify(items, null, 4))
```

But is a little bit inconvenient that every time I run it, a new log window is opened.

Is there a way to check if a log window is already opened for the current script, and if so, focus it instead of opening a new one?

Or maybe I should use something like editor to see the result in a disposable text editor view 🤔

**Comment by johnlindquist** [comment]
Oh, that's a good point. I'll make sure `showLogWindow()` doesn't open a second window in the next build (probably tomorrow)
